mappings/net/minecraft/obfuscate/DontObfuscate.mapping

```diff
@@ -0,0 +1,25 @@
+CLASS net/minecraft/obfuscate/DontObfuscate net/minecraft/obfuscate/DontObfuscate
+	COMMENT This annotation applies to some unobfuscated elements within the Minecraft
+	COMMENT source code.
+	COMMENT
+	COMMENT <p>Its behavior appears as follows:
+	COMMENT <ul>
+	COMMENT <li>The annotation itself is not obfuscated.</li>
+	COMMENT <li>If a class is annotated, it is not obfuscated.</li>
```

Comment:
Does it make all members not obfuscated? Should probably clarify that.

Reply:
I don't know. In the only case it's used in a class, that class is used a functional interface whose member won't be obfuscated in any case.